pkgs/development/libraries/openmpi/default.nix

```diff
@@ -29,6 +33,14 @@ in stdenv.mkDerivation rec {
     sha256 = "0ms0zvyxyy3pnx9qwib6zaljyp2b3ixny64xvq3czv3jpr8zf2wh";
   };
 
+  # TODO: Remove these, when a new release gets out
+  # https://github.com/open-mpi/ompi/commit/526775dfd7ad75c308532784de4fb3ffed25458f.patch
+  # https://github.com/open-mpi/ompi/commit/a3026c016a6a8be379f62585b6ddc070175c8106.patch
+  patches = [
+    ./ucx-1.8.patch
+    ./ucx-1.7.patch
```

Comment:
According to [upstream UCX](https://github.com/openucx/ucx/wiki/OpenMPI-and-OpenSHMEM-installation-with-UCX#running-open-mpi-with-ucx), configuring openmpi with `--enable-mca-no-build=btl-uct` solves the problem with no impact on performance.

Reply:
i usually prefer to build the package as a whole, to give all options to users - but i've tested `--enable-mca-no-build=btl-uct` and it also works that way.

Reply:
The solution to disable the uct btl seems to be more simple thus I would prefer that one (unless there is a motivation to activate it).

Reply:
should be resolved with the `fetchpatch` approach from above and can be removed when the next openmpi release lands.